cdap-api/src/main/java/co/cask/cdap/api/customaction/CustomActionContext.java

```diff
@@ -25,7 +25,7 @@
 /**
  * Represents runtime context of the {@link CustomAction} in the Workflow.
  */
-public interface CustomActionContext extends RuntimeContext, Transactional, PluginContext, WorkflowInfoProvider {
+public interface CustomActionContext extends RuntimeContext, Transactional, WorkflowInfoProvider {
```

Comment:
why is PluginContext getting removed? Without it, we cannot use it in apps like Hydrator.